tests/cache-functions.spec.ts

```diff
@@ -127,7 +127,7 @@ describe('getCacheKey', () => {
         };        
 
         let cacheKeyStr = `v=${version}${cacheSuffix}::cmake=${testEnv.CMAKE_VERSION}::options=::${repo}=${sha}`;
-        for (const [dependency, dependencySha] of Object.entries(testEnv.DEPENDENCIES || {})) {
+        for (const [dependency, dependencySha] of Object.entries(env.DEPENDENCIES || {}).sort((a, b) => a[0] > b[0] ? 1 : -1)) {
```

Comment:
`testEnv` has changed to `env` here - presumably a typo?

Why didn't the test fail because of this?